README.md

```diff
@@ -7,7 +7,7 @@
 
 The SCM is a tool for developing physics suites and diagnosing their performance. This SCM works with the
 - [Common Community Physics Package (CCPP):](https://github.com/NCAR/ccpp-physics) a library of physical parameterizations for atmospheric numerical models.
-- [CCPP Framework:](https://github.com/NCAR/ccpp-scm:) a framework for connecting atmospheric models to physics suites.
+- [CCPP Framework:](https://github.com/NCAR/ccpp-scm) a framework for connecting atmospheric models to physics suites.
```

Comment:
I think the link should actually be https://github.com/NCAR/ccpp-framework

Reply:
The evils of copy-and-paste, thanks for pointing this out!